app/articles/views.py

```diff
@@ -118,6 +119,13 @@ class ArticleUnFavouriteView(generics.UpdateAPIView):
     renderer_classes = (JSONRenderer,)
 
 
+class ArticleStatsView(generics.ListCreateAPIView):
```

Comment:
why are you using ListCreate instead of ListAPIView?

Reply:
Initially I had a queryset